ProcessManager.h

```diff
@@ -48,7 +48,7 @@ class ProcessManager
 	long contentLongInt;
 	///Store the data read from the memory addresses akin to how the data is stored in variables in a standard programming language.
 	///stores long int/memory address (*which in turn will make the BaseAddress as a pointer to this memory address*)
-	long contentLongHex;
+	//long contentLongHex;
 	///Store the data read from the memory addresses akin to how the data is stored in variables in a standard programming language.
 	///stores the char interpretation of data
 	char contentChar;
```

Comment:
I think lines 49 and 50 will be treated as documentation by doxygen for code on line 54. Is it?

Reply:
@ssn7i respond on this so we can resolve and merge.

Reply:
deleting the line is also fine, I just commented it due to laziness. (Won't be picked up by Doxygen since I've only used double (not triple) slash for commenting.